src/generator/backend/assets/css/schedule.css

```diff
@@ -0,0 +1,249 @@
+body {
+  font-family: 'Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
+  color: #777777;
+}
+
+.h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
+  font-family: 'Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
+  color: #333333;
+  font-weight: 300;
+}
+
+a {
+  color: #777;
+}
+
+a:hover {
+  color: #e12b00;
+  text-decoration: none;
+}
+
+/* Social media icons for speakers */
+a.speaker-social {
+  margin-right: 0.3em;
+}
+
+a.speaker-social:hover {
+  text-decoration: none;
+}
+
+/* Navbar adjustments */
+.logo {
+  max-width: 110px;
+  max-height: 45px;
+}
+
+.navbar-toggle {
+  padding: 6px 10px;
+  color: #777;
+}
+
+/* General styling of session list */
+.session-list li {
+  cursor: pointer;
+}
+
+.session-list .label {
+  background: #ff8700;
+  border-color: #ebccd1;
+  color: #FFFFFF;
+  font-weight: 500;
+}
+
+.session-list .session-title {
+  margin-top: 0.1em;
+}
+
+.session-list .session-title .session-link {
+  font-size: 12px;
+}
+
+/* Adjustments for smaller screens */
+body {
+  font-size: 12px;
+}
+
+.session-list .label {
+  font-size: 80%;
+  margin-left: -8px;
+}
+
+.session-location {
+  text-align: left;
+}
+
+@media (min-width: 450px) {
+  body {
+    font-size: 14px
+  }
+
+  .session-list .label {
+    font-size: 90%;
+    padding: .2em .6em .3em;
+  }
+
+  .session-title {
+    float:left;
+    width: 75%;
+  }
+
+  .session-location {
+    float: right;
+    width: 25%;
+    text-align: right;
+  }
+}
+@media (min-width: 768px) {
+
+    #day-menu .dropdown-menu {
+        display: block;
+        opacity: 0;
+        top: -500px;
+        -moz-transition: opacity 500ms ease;
+        -webkit-transition: opacity 500ms ease;
+        -o-transition: opacity 500ms ease;
+        -ms-transition: opacity 500ms ease;
+        transition: opacity 500ms ease;
+    }
+
+    #day-menu > .dropdown-toggle:hover + .dropdown-menu {
+        display: block;
+        opacity: 1;
+        top: 50px;
+    }
+
+    #day-menu > .dropdown-toggle:hover {
+        background-color: #e7e7e7;
+    }
+
+    #day-menu .dropdown-menu:hover {
+        display: block;
+        opacity: 1;
+        top: 50px;
+    }
+}
+.btn-sign-up {
+  background: #ff8700;
+  color: #FFFFFF;
+  margin-left: 5px;
+  margin-top: -5px;
+}
+
+.video-link > .fa {
+	font-size: 22px;
+	color: #757575;
+}
+
+.video-link > .fa:hover {
+	color: #e52d27;
+}
+
+.video-link  {
+	margin-left: 3px;
+}
+
+.session-link > .fa {
+	font-size: 18px;
+	color: #757575;
+}
+
+.session-link > .fa:hover {
+	color: #2196F3;
+}
+
+.session-link + .inputbox {
+  font-size: 12px;
+}
+
+@media (max-width: 450px) {
+	.session-link > .fa {
+		font-size: 24px;
+	}
+	.video-link > .fa {
+		font-size: 28px;
+	}
+	.btn-sign-up {
+		font-size: 15px;
+		margin-top: -8px;
+	}
+	.session-title {
+		line-height: 1.5;
+	}
+}
+@media (max-width: 500px) {
+  .nav>li>a {
+    padding:8px !important;
+  }
+}
+.inputbox {
+  width:35%;;
+  padding: 4px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using width with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710180111/issues/source?bid=3233857&fileBranchId=3342834#l193)